core/tests/widgets/test_imageview.py

```diff
@@ -113,26 +126,26 @@ def test_set_image_none(app):
     "params, expected_width, expected_height, expected_aspect_ratio",
     [
         # Intrinsic image size
-        (dict(style=Pack()), 60, 40, 1.5),
-        (dict(style=Pack(), scale=2), 120, 80, 1.5),
+        (dict(style=Pack()), 32, 32, 1),
```

Comment:
This makes me a little nervous; the aspect ratio test was helpful because it verified we weren't returning height as width (or vice versa). The testbed app has a sample image with a 2:1 aspect ratio, and the core tests folder already has a collection of resources; I suspect we may need a copy of that image in the core tests.